Makefile

```diff
@@ -29,6 +29,8 @@ KOPANO_WEBAPP_SMIME_REPOSITORY_URL := file:/kopano/repo/smime
 KOPANO_ZPUSH_REPOSITORY_URL := http://repo.z-hub.io/z-push:/final/Debian_9.0/
 RELEASE_KEY_DOWNLOAD := 0
 DOWNLOAD_COMMUNITY_PACKAGES := 1
+KOPANO_UID=999
```

Comment:
This should be `KOPANO_UID := 999` to only define it if it was not already set in the environment. Same for the GID variable